### PR TITLE
{data}[GCCcore/11.2.0] pugixml v1.12.1

### PR DESCRIPTION
**Pull request by sassy-crick**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/p/pugixml/pugixml-1.12.1-GCCcore-11.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+checksums = [
+    '1e28ab24b6e04e013d96f45d25e9f2d04c921dc68c613fd010ecaaad3892c14d',  # v1.12.1.tar.gz
+]
+
+builddependencies = [
+    ('binutils', '2.37'),
+    ('CMake', '3.22.1')
+]
+
+configopts = '-DBUILD_SHARED_LIBS=ON -DPUGIXML_BUILD_SHARED_AND_STATIC_LIBS=ON -DPUGIXML_HEADER_ONLY=True '
```
Comment (bedroge) [inline easybuild/easyconfigs/p/pugixml/pugixml-1.12.1-GCCcore-11.2.0.eb:24]: The older easyconfig already contains this, but I don't think this `-DPUGIXML_HEADER_ONLY=True` is actually doing anything. In my build log I see:
```
CMake Warning:
  Manually-specified variables were not used by the project:

    CMAKE_C_COMPILER
    CMAKE_C_FLAGS
    CMAKE_FIND_USE_PACKAGE_REGISTRY
    CMAKE_Fortran_COMPILER
    CMAKE_Fortran_FLAGS
    PUGIXML_HEADER_ONLY
```
And I don't see any difference when building with/without this option. The documentation says you have to define this `PUGIXML_HEADER_ONLY` in your " pugiconfig.hpp, or provide them via compiler command-line", so it's not actually a CMake variable.

Hence, I guess we could/should just remove this option here?

```suggestion
configopts = '-DBUILD_SHARED_LIBS=ON -DPUGIXML_BUILD_SHARED_AND_STATIC_LIBS=ON '
```

Comment (sassy-crick) [reply easybuild/easyconfigs/p/pugixml/pugixml-1.12.1-GCCcore-11.2.0.eb]: Yes, you are right. I have changed that. Thanks for pointing that out, much appreciated.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3100.skitty.os - Linux RHEL 8.4, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/a0fadbdb3d9a67e654f5150f34eff0c2 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @sassy-crick!

**Pull request merged**